patches/configs/310.config

```diff
@@ -1,8 +1,7 @@
 [build_id]
-main=034C8FA7A63B7A87F96F408B2AEFFF6C
-sdk=EAD5AAD43C3EFA01682460ECE70A8189F84D2AFF
+main=3CA12DFAAF9C82DA
```

Comment:
`3CA12DFAAF9C82DA` is the Build ID for Super Mario Odyssey v1.0.0.

This is a fairly well-known string, and you can likely find the ID for other versions by searching online; but EdiZon is also a good option for finding it yourself (you can do so through the Cheats menu, as of writing).

Reply:
Update: Atmosphère actually requires a longer form of ID here. I've edited this commit after-the-fact to use `3CA12DFAAF9C82DA064D1698DF79CDA1`.

Reply:
The longer form of the build id can be found in the `main` module. The 16 byte value is located at `0x40` - `0x4F`